lib/payload/scanner/kippo_honeypot/engine.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+import sys
+import socket
+
+def recv_all(s, limit=4196):
```

Comment:
you may use `from lib.payload.scanner.service.engine import recv_all` to prevent duplicated code, I should mention I used duplicate in `ics_honeypot` because it's standalone and run from another directory, but since it's a module please use `from lib.payload.scanner.service.engine import recv_all`